journal-writer/src/main/scala/akka/persistence/journal/writer/WriteJournalAdapterCameo.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.persistence.journal.writer
```

Comment:
this must be in the `akka.persistence` package because it is using internal API?